pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/bootstrap-sources.nix

```diff
@@ -85,22 +85,13 @@ rec {
       #
       # Neither your store nor your substituters seems to have:
       #
-      #  ${name}.nar.xz
+      #  /nix/store/________________________________-${name}
       #
-      # Please obtain or create this file, give it exactly the name
-      # shown above, and then run the following command:
-      #
-      #   nix-store --add-fixed ${outputHashAlgo} ${name}.nar.xz
-      #
-      # You can create this file from an already-bootstrapped nixpkgs
+      # You can create this path from an already-bootstrapped nixpkgs
       # using the following command:
       #
       #   nix-build '<nixpkgs>' -A make-minimal-bootstrap-sources
       #
-      # Or, if you prefer, you can create this file using only `git`,
-      # `nix`, and `xz`.  For the commands needed in order to do this,
-      # see `make-bootstrap-sources.nix`.
-      #
```

Comment:
Being able to give the user these instructions is kinda important.  Let me see if I can find a way to get back to that point.

Reply:
We can have the same instructions for doing it manually with the other Nix store add command, no?

Reply:
That's what I'm going to try to figure out.

Reply:
I added them bakc.